carbonmark/components/Transaction/index.tsx

```diff
@@ -21,17 +20,25 @@ interface Props {
   approvalText?: React.ReactNode;
   submitText?: React.ReactNode;
   onGoBack?: () => void;
+  spenderAddress: string;
+  tokenIcon?: StaticImageData;
+  tokenName?: string;
+  onViewChange?: (newView: "approve" | "submit") => void;
+  showPrice?: boolean;
```

Comment:
You can remove this prop as mentioned above.
Either a `price` is passed or not.

Also you are not using `showPrice` further down.